pkg/filesystem/virtual/pool_backed_file_allocator.go

```diff
@@ -332,6 +342,8 @@ func (f *fileBackedFile) virtualGetAttributesLocked(attributes *Attributes) {
 	}
 	attributes.SetPermissions(permissions)
 	attributes.SetSizeBytes(f.size)
+	attributes.SetOwnerUserID(f.ownerUserID)
+	attributes.SetOwnerGroupID(f.ownerGroupID)
```

Comment:
You can omit this. All calls to `virtualGetAttributesLocked()` are preceded by `virtualGetAttributesUnlocked()`.